rts/Rml/Backends/RmlUi_Renderer_GL3_Spring.cpp

```diff
@@ -657,6 +657,10 @@ bool RenderInterface_GL3_Spring::GenerateTexture(Rml::TextureHandle& texture_han
 
 void RenderInterface_GL3_Spring::ReleaseTexture(Rml::TextureHandle texture_handle)
 {
+	// Something was using a texture loaded/managed outside of Rml. Do nothing.
```

Comment:
Sounds worrisome, when can this happen? What if "the outside" frees the texture while RML is still using it?

Reply:
Then you end up getting a broken texture when you try to render.
Same thing that can happen if RML frees up a texture that's still in use elsewhere.

I think it's a better solution than copying the texture data needlessly. Though I suppose someone could force a copy to happen.

This check was added because of the <lua-texture> element which asks for texture handles from `LuaOpenGLUtils::ParseTextureImage`.
Which I made because I think loading in another copy of all the buildpics/icons for use in by RML is a terrible idea.

Reply:
A copy texture function could also be made- for when you aren't certain that the original texture handle is going to stick around when you need it to?